contributing/deployment-tools.txt

```diff
@@ -0,0 +1,147 @@
+OME deployment tools
+====================
+
+This section describes deployment tools supported by the OME team.
+It is primarily designed for the core OME developers who want to
+bring new or upgrade existing prerequisites. The following steps explain
+the connections between basic repositories and the testing workflow.
+
+
+.. note::
+
+    This section requires brief understanding of ansible 
+    http://docs.ansible.com/ansible/intro_getting_started.html
+    and Docker containers https://docs.docker.com/.
+
+Prerequisites locations
+-----------------------
+
+List of OME prerequisites is stored in multiple git repositories, each of 
+which is available from several locations.
+
+OME Infrastructure (Experimental)
```

Comment:
Given the effort put under this repo and its usage for a prod instance like IDR, I would think `experimental` is an understatement.